BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -53,9 +56,6 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
             title: store.state.mode.navigationTitle,
             titleDisplayMode: .inline
         )
-        .task {
-            await store.perform(.loadData)
```

Comment:
noticed this was being called twice.  I can add it back but wanted to call it out in case it was missed before. CC: @matt-livefront